### PR TITLE
added ab_elementwise_op support into splitK Gemm

### PR DESCRIPTION
**Pull request by zjing14**

### CODE REVIEW

File: include/ck/tensor_operation/gpu/device/impl/device_gemm_xdl_splitk_c_shuffle.hpp [hunk 1]
```diff
@@ -126,7 +126,51 @@ struct DeviceGemmXdlSplitKCShuffle : public DeviceGemmSplitK<ALayout,
...
+              c_element_op(c_element_op_)
+        {
+        }
+
+        AElementwiseOperation a_element_op;
+        BElementwiseOperation b_element_op;
+        CElementwiseOperation c_element_op;
+    };
+
+    // using Argument              = typename GridwiseGemm::Argument;
```
Comment (aosewski) [inline include/ck/tensor_operation/gpu/device/impl/device_gemm_xdl_splitk_c_shuffle.hpp]: Leftover.

File: include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_v2r4r2.hpp [hunk 2]
```diff
@@ -44,6 +44,40 @@ __global__ void
...
+          InMemoryDataOperationEnum CGlobalMemoryDataOperation,
+          typename Block2CTileMap,
+          typename AElementwiseOperation,
+          typename BElementwiseOperation,
+          typename CElementwiseOperation>
+__global__ void
+#if CK_USE_LAUNCH_BOUNDS
+    __launch_bounds__(CK_MAX_THREAD_PER_BLOCK, CK_MIN_BLOCK_PER_CU)
+#endif
+        kernel_gemm_xdlops_v2r4r2_simplified(typename GridwiseGemm::Argument karg,
```
Comment (aosewski) [inline include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_v2r4r2.hpp:33]: Do we still need the version without ABC elementwise ops above [here](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/956/files#diff-325efd84547132f6a10cb99736c02a47cc837d12b51de9cb33ab04f1c4211030R30)?

File: include/ck/tensor_operation/gpu/grid/gridwise_gemm_multiple_abd_xdl_cshuffle.hpp [hunk 3]
```diff
@@ -657,6 +657,7 @@ struct GridwiseGemmMultipleABD_xdl_cshuffle
         auto blockwise_gemm = BlockwiseGemmXdlops_k0mk1_k0nk1_m0n0m1n1m2m3m4n2_Selector<
             BlockSize,
             ComputeDataType,
+            ComputeDataType,
```
Comment (aosewski) [inline include/ck/tensor_operation/gpu/grid/gridwise_gemm_multiple_abd_xdl_cshuffle.hpp]: Was this a bug? Is this change needed here? It's not related to the PR's title.

Comment (zjing14) [reply include/ck/tensor_operation/gpu/grid/gridwise_gemm_multiple_abd_xdl_cshuffle.hpp]: This is not a bug. We allow ComputeType for A/B is different.

Comment (aosewski) [reply include/ck/tensor_operation/gpu/grid/gridwise_gemm_multiple_abd_xdl_cshuffle.hpp]: Well, but there seemed to be missing one compute type argument. Good you caught that :)

**Review by aosewski — DISMISSED** [review]

**Review by aosewski — APPROVED** [review]
